### PR TITLE
fix issue with urfave/cli

### PR DESCRIPTION
**Pull request by ubogdan**
**Describe the PR**
Fix urfave/cli dependecy

**Relation issue**
#556 

**Additional context**
None.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -52,17 +53,25 @@ clean:
...
-		golang.org/x/lint/golint \
-		github.com/swaggo/swag/cmd/swag	\
-		github.com/swaggo/swag/gen
+		golang.org/x/lint/golint 
+
+.PHONY: ensure-gopath
+ensure-gopath:
+	mkdir -p ${GOPATH}/github.com/swaggo
+	if [ -L ${GOPATH}/github.com/swaggo/swag ]; then rm ${GOPATH}/github.com/swaggo/swag; fi
+	ln -s "$(shell pwd)"  ${GOPATH}/github.com/swaggo/swag
```
Comment (easonlin404) [inline Makefile]: I wonder why there needs to add `ensure-gopath` to check?

Comment (ubogdan) [reply Makefile]: ensure-gopath is only for go1.10

Comment (ubogdan) [reply Makefile]: If you don't like it there please let me know where I should put it.

Comment (easonlin404) [reply Makefile]: no worry. Just a question. 👍

**Pull request merged**